src/Ledger/Utxow.lagda

```diff
@@ -186,4 +195,24 @@ pattern UTXOW⇒UTXO x = UTXOW-inductive⋯ _ _ _ _ _ _ _ _ x
 unquoteDecl UTXOW-inductive-premises =
   genPremises UTXOW-inductive-premises (quote UTXOW-inductive)
 \end{code}
-\end{NoConway}
+
+\href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-0069}{CIP-0069}
+unifies the arguments given to all types of Plutus scripts currently available
+(spending, certifying, rewarding, minting) by removing the argument of a datum.
+This aims to address the mutual dependency issue (two validators that need to
+know each other's hash), which arises when designing dapps and is widely
+considered a substantial barrier to safe protocols and a considerable limitation
+on our design space.
+
+According to CIP-0069, every validator must take the redeemer and script
```

Comment:
This is already in Alonzo, nothing new.

Reply:
So, delete line 207?  (I don't see the harm in reiterating an explanation, even if it was explained previously, in another document.)